numpy/_core/src/multiarray/methods.c

```diff
@@ -892,24 +892,24 @@ static PyObject *
 array_wraparray(PyArrayObject *self, PyObject *args)
 {
     PyArrayObject *arr;
-    PyObject *obj;
+    PyObject *context_ignored = NULL;
```

Comment:
Had to do a slight double take of whether this was a flag for whether the context would be ignored.  How about just 
```
PyObject *UNUSED = NULL;  # for the context argument
```